README.md

```diff
@@ -1,5 +1,4 @@
-mapbox-gl-compare
----
+## mapbox-gl-compare
```

Comment:
Why this change? 

```md
mapbox-gl-compare
---
```

should effectively be the same thing

Reply:
No idea - and I think Prettier made that change - it's reverted in next commit